src/tests/ftest/rebuild/pool_destroy_rebuild.yaml

```diff
@@ -0,0 +1,63 @@
+# change host names to your reserved nodes, the
+# required quantity is indicated by the placeholders
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+  test_clients:
+    - client-C
+timeout: 180
+server_config:
+  servers_per_host: 2
+  name: daos_server
+  servers:
+    0:
+      targets: 4
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      bdev_class: nvme
```

Comment:
If it is possible to remove the nvme config on this test? If nvme is not needed in the rebuild test.

Reply:
Any specific reason you want me to remove nvme? Whole intention is, if I am using hardware nodes anyways then why not make use of all the perks offered by it.

Reply:
Only a small number of hosts has nvme, we should remove the dependency to allow the rebuild functional test launch easier. 
On the rebuild functional test-dir, we should have all the scripts consist with similar test-environments and requirement.